src/Redirection/Dynamic/RedirectionManager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace AppBundle\Redirection\Dynamic;
+
+use AppBundle\Entity\Redirection;
+use Doctrine\ORM\EntityManagerInterface;
+use JMS\Serializer\Serializer;
+use Symfony\Component\Cache\Adapter\AdapterInterface;
+
+class RedirectionManager
+{
+    private $cache;
+    private $entityManager;
+    private $serializer;
+
+    public function __construct(AdapterInterface $cache, EntityManagerInterface $entityManager, Serializer $serializer)
+    {
+        $this->cache = $cache;
+        $this->entityManager = $entityManager;
+        $this->serializer = $serializer;
+    }
+
+    public function refreshRedirectionCache(Redirection $redirection): void
+    {
+        $item = $this->cache
+            ->getItem('redirection-'.md5($redirection->getFrom()))
+            ->set($this->serializer->serialize($redirection, 'json'))
```

Comment:
What do you think if we only serialize the needed information to perform the redirection ? Instead of the whole object ?

Reply:
In the object we have 3/5 attributes used for the redirection. I can configure a serialization group if you think it's important.

Reply:
Yeap it would be more accurate 👍